packages/presets/src/scripts/deploy-app.ts

```diff
@@ -64,6 +64,7 @@ export async function deployApp() {
     await uploadFile({ deployer, fileContract, fileContents, filename, partitions, isLive, application });
   }
 
+  console.log(`your application is now available at http://ec2-54-185-81-147.us-west-2.compute.amazonaws.com/${fileContract}/${deployerAccount}/${application}`)
```

Comment:
Can we somehow make `http://ec2-54-185-81-147.us-west-2.compute.amazonaws.com` dynamic? Just in case this is about to be updated.